### ISSUE TITLE
[MU4 Issue] Dump files missing on Mac

### ISSUE THREAD

**Issue by DmitryArefiev** [musescore/MuseScore#13387]
**Describe the bug**
Dump .dmp files missing on Mac

**To Reproduce**
1. Go to ~/Library/Application Support/MuseScore/MuseScore4Development/logs/dumps

On **Mac**
![Screenshot 2022-09-16 at 12 33 07](https://user-images.githubusercontent.com/90187801/190671955-b8b58534-0e94-4f29-a03e-56d4dd0d9e25.png)

On **Win**

![Capture2](https://user-images.githubusercontent.com/90187801/190672023-a998fdfa-812b-4bfb-bdcc-bd2e7de85be7.PNG)

**Expected behavior**

**Screenshots**

**Platform information**
macOS 12.6

**Comment by bkunda** [comment]
@Eism Is this something you are working on? Pls advise whether it is achievable to fix for 4.1. Thanks!

**Comment by Eism** [comment]
No, that's another issue that we have decided to address in the upcoming versions. Unfortunately, fixing it now is challenging. 
Currently, crash reports only work on x86 processors on macOS and does not work for ARM (M1) processors.

**Comment by cbjeukendrup** [comment]
Is this still actual?

**Comment by DmitryArefiev** [comment]
Tested with **4.6.2**, **4.6.3 nightly** and **4.7 nightly** on Mac13.7.8 (M1 MacBook Air). **FIXED**

The only thing, in developers builds the dmp files are stored in **pending** folder and not replaced to **completed** folder (FYI @Eism )
